resources/tests/compilerTests/testFFI.obs

```diff
@@ -0,0 +1,9 @@
+import "Obsidian_Runtime/src/main/java/Runtime/edu/cmu/cs/obsidian/stdlib/IO.obs"
```

Comment:
Rename this file to TestFFI and change contract name to match, just to be consistent with the other tests.